include/qthread/qthread.hpp

```diff
@@ -14,7 +14,11 @@
 template <bool> class OnlyTrue;
 template <> class OnlyTrue<true>
 {};
+#if __cplusplus >= 201103
```

Comment:
One nice thing about that test is that if `__cplusplus` is undefined, then the preprocessor has to treat it a zero, therefore it causes the test to do the right thing.